index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+    </section>
+
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+
+    <section class="container about_container">
```

Comment:
- [ ] Kindly adjust the background image to look like the Figma design 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_16](https://user-images.githubusercontent.com/69638013/159150946-aa588c7f-cc27-4a51-b355-eec8a98fc64e.png) |  ![hehehe](https://user-images.githubusercontent.com/69638013/159150942-7dfef570-c0c6-483d-94eb-2572d137734d.png)

- [ ] Your project is missing the border before languages please  add it 👍🏿